flow/embedded_views.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+//
+#ifndef FLUTTER_FLOW_EMBEDDED_VIEWS_H_
+#define FLUTTER_FLOW_EMBEDDED_VIEWS_H_
+
+#include "flutter/fml/memory/ref_counted.h"
+
+namespace flow {
+
+class EmbeddedViewParams {
+ public:
+  double x;
```

Comment:
Please use SkPoint and SkSize with appropriately descriptive variable names instead. It is clear `x` and `y` are members of a point, but what does that point denote?

Reply:
In the context of this PR this is just a stub. (I wanted to keep the plumbing noise away from any other work)

I'm changing to SkPoint and SkSize in #6614